src/common/MovieTile/styled.js

```diff
@@ -0,0 +1,150 @@
+import styled from "styled-components";
+import { ReactComponent as YellowStar } from "./images/shape-star.svg";
+
+export const Wrapper = styled.div`
+    max-width: 324px;
+    height: 650px;
+    padding: 16px;
+    gap: 16px;
+    justify-content: left;
+    background: ${({ theme }) => theme.colors.white};   
+    box-shadow: ${({ theme }) => theme.common.boxShadow};  
+
+    @media (max-width : ${({ theme }) => theme.common.breakpoint}) {
+        max-width: 100%;
+        height: 201px;
+        align-items: center;
+        display: flex;
+        flex-flow: nowrap; 
+    }
+`;
+
+export const Poster = styled.div`
+    width: 292px;
+    height: 434px;
+    border-radius: 5px;
+    background-color: green; // it's temporary - here will be poster from API //
+
+    @media (max-width : ${({ theme }) => theme.common.breakpoint}) {
+        width: 114px;
+        height: 169px;
+    }
+`;
+
+export const DescriptionBox = styled.div`
+    width: 292px;
+    height: 200px;
+    position: relative; 
+    padding-top: inherit;
+    padding-bottom: inherit;
+
+    @media (max-width : ${({ theme }) => theme.common.breakpoint}) {
+        width: auto;
+        position: inherit;
+    }
+`;
+
+export const Title = styled.span`
+    font-size: 22px;
+    font-style: normal;
```

Comment:
I don't think there is any point of setting `font-style` to normal here. Not only its a default value of the website, but you're also overwriting it with oncoming `font-weight` below 😅 (same thing in the other styled components).

Reply:
You got me ! :D

Reply:
Done !